ccd-definition/CaseField.json

```diff
@@ -251,6 +251,23 @@
     "FieldType": "Respondent",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "respondents1",
+    "Label": "Respondent",
+    "FieldType": "Collection",
+    "FieldTypeParameter": "partyRespondent",
```

Comment:
Let's remember to rename complex type back to `Respondent` when we removed currently used complex type that holds first* / additional*.

Reply:
yep - so there's a second step to this migration. Maybe a conversation we need to have, I am not sure how we will remove respondents completely. Might have to go as far as remove ALL data and rebuild. I don't think .remove on the data map will remove it (I mean we need to set it to empty currently)